src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.scss

```diff
@@ -0,0 +1,30 @@
+@import 'src/main/webapp/content/scss/artemis-variables';
+
+jhi-collapsable-assessment-instructions {
```

Comment:
As far as I checked via inspecting this element, the max-height is not actually set. I guess, you can remove this here, the error will then also disappear.

Reply:
No, I cannot remove it because `max-height` is actually used. @lschlesinger 
_In the 1st screenshot_: You can notice where the bottom line ends. The container is a scrollable view. 
_In the second screenshot_: The container is not scrollable anymore because its height became big enough for all the information to fit in.

### With max-height defined (1st sceenshot)
![withheight](https://user-images.githubusercontent.com/51077603/125211943-5860f900-e2aa-11eb-94bf-dec1e1168945.jpeg)

### Without max-height (2nd screenshot)
![without](https://user-images.githubusercontent.com/51077603/125211942-55660880-e2aa-11eb-9099-248f3a812835.jpeg)

Reply:
ok, I might have overlooked something. Sorry.